src/content/en/fundamentals/performance/audit/index.md

```diff
@@ -58,14 +58,18 @@ against peers and estimate the potential revenue opportunity of improving site s
 
 Make sure to test your website as a
 first-time user sees it. Open the site an Incognito (Private) Window, or use browser tools to
-disable caching and clear storage. This ensures that every asset is retrieved from the network and not
-from a local cache, so you get an accurate picture of first-load performance.
+disable caching and clear storage. This ensures that every asset is retrieved from the 
+network and not from a local cache, so you get an accurate picture of first-load performance.
 
-<div class="note"><a href="(https://www.webpagetest.org">WebPagetest</a> is a free service that runs performance tests from multiple locations globally using real browsers with real connections, to provide detailed metrics and optimization advice.
+<div class="note"><a href="(https://www.webpagetest.org">WebPagetest</a> is a free service that 
+runs performance tests from multiple locations globally using real browsers with real connections, 
+to provide detailed metrics and optimization advice.
   <ul>
-    <li>Leverage <a href="(https://www.webpagetest.org/easy">webpagetest.org/easy</a> to quickly simulate differences between network connection types on mobile</li>
+    <li>Leverage <a href="(https://www.webpagetest.org/easy">webpagetest.org/easy</a> to quickly 
+simulate differences between network connection types on mobile</li>
     <li>Generate a Lighthouse audit and report with every WebPagetest trace.</li>
-    <li> Measure site on first visit as well as for repeat views — for example: to see how much service worker caching helps.</li>
+    <li> Measure site on first visit as well as for repeat views — for example: to see how much 
+service worker caching helps.</li>
```

Comment:
indentation should match please

Reply:
Do you mean I should indent all follow-on lines that I've had to manually wrap to fit 80 characters?

Reply:
Yes, correct.

Reply:
OK — done.